.rubocop.yml

```diff
@@ -4,7 +4,7 @@ require:
   - rubocop-rake
   - rubocop-rspec
 AllCops:
-  TargetRubyVersion: 2.4
+  TargetRubyVersion: 2.5
```

Comment:
Set to 2.5 because we still maybe want to remain compatible to 2.5 at least for a while